CGPs/0003.md

```diff
@@ -0,0 +1,73 @@
+# CGP [0003]: Rename Celo Gold to Celo Native Asset
+
+- Date: 2020-06-04
+- Author(s): @JamesCowling
+- Status: DRAFT
+- Governance Proposal ID #: 3
+- Date Executed: [if executed]
+
+## Overview
+
+This proposal decrees that the name "Celo Gold" be discontinued and that the
+Celo native asset be officially referred to as simply the "Celo native asset".
+The shortened name Celo (CELO) will be used as ticker symbol and in casual
+parlance.
+
+The name Celo Gold could be confusing since it doesn't relate to the price of
+gold. There will likely be multiple Celo stablecoins with names like Celo
+Dollar, Celo Euro and Celo Peso one day and potentially even a Celo Gold
+stablecoin.
+
+Even despite the potential naming collision, the gold metaphor doesn't
+adequately represent the properties of the native Celo asset, which include
+staking and voting on governance.
+
+The name "Celo" casts a clear distinction between the native asset and the
+suite of stablecoins that will develop over time.
+
+It is important to establish appropriate naming now while it's early in the
+life of the ecosystem.
+
+## Proposed Changes
+
+This proposal does not dictate specific transactions or code changes since these
+will have to roll out over time. It instead establishes the new naming
+in principal and can be subsequently acted upon by Celo contributors. These
+changes will span the Celo documentation and marketing, as well as the Celo
+codebase where appropriate.
+
+Note that it will likely not be feasible to exhaustively remove all references
+to Gold in the Celo codebase. This is deemed as acceptable given that this
+complexity is levied on developers who are familiar with the ecosystem and not
+the general public.
+
```

Comment:
Presumably, we would need to also update the ticker entry for the BIP-0044 derivation path here: https://github.com/satoshilabs/slips/blob/master/slip-0044.md